src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -180,6 +180,7 @@ public CheckResult SetBindingInfo(RecordType parameterType)
 
         private FormulaType _expectedReturnType;
         private bool _allowCoerceToType = false;
+        private FormulaType[] _allowedInputTypes;
 
         public CheckResult SetExpectedReturnValue(FormulaType type, bool allowCoerceTo = false)

```

Comment:
>SetExpectedReturnValue [](http://example.com/codeflow?start=27&length=22)

Can we add a comment that this only injects errors, but does not add any coercions. So caller still has to do the coercions.

Reply:
Added comment.